src/test/resources/alma/(DE-605)HT017015300.json

```diff
@@ -181,6 +181,10 @@
     "id" : "http://digitale-objekte.hbz-nrw.de/storage/2012/06/16/file_46/4507767.pdf",
     "label" : "Inhaltsverzeichnis"
   } ],
+  "related" : [ {
+    "isbn" : [ "384033490X", "9783840334900" ],
+    "note" : "Erscheint auch als Online-Ausgabe"
+  } ],
```

Comment:
up to today we have only configured related.id and related.label. (see index-config.json). We might want to use more fields. Note that both fields won't be queryable, which might be ok.

Reply:
You are right I added isbn and note since not all linked ressources have a hbz ID. In this way we can at least state if they have.